docs/section-2-using-parsers.md

```diff
@@ -629,18 +629,36 @@ The restrictions placed on a pattern by an anchor operator ignore anonymous node
 
 #### Predicates
 
-You can also specify arbitrary metadata and conditions associated with a pattern by adding _predicate_ S-expressions anywhere within your pattern. Predicate S-expressions start with a _predicate name_ beginning with a `#` character. After that, they can contain an arbitrary number of `@`-prefixed capture names or strings.
+You can also specify arbitrary metadata and conditions associated with a pattern
+by adding _predicate_ S-expressions anywhere within your pattern. Predicate S-expressions
+start with a _predicate name_ beginning with a `#` character. After that, they can
+contain an arbitrary number of `@`-prefixed capture names or strings.
 
-For example, this pattern would match identifier whose names is written in `SCREAMING_SNAKE_CASE`:
+Tree-Sitter's CLI supports the following predicates by default:
+
+##### eq?, not-eq?, any-eq?, any-not-eq?
+
+This family of predicates allows you to match against a single capture or string
+value.
+
+The first argument must be a capture, but the second can be either a capture to
+compare the two captures' text, or a string to compare first capture's text
+against.
+
+The base predicate is "#eq?", but its complement "#not-eq?" can be used to _not_
+match a value.
+
+Consider the following example targeting C:
 
 ```scheme
-(
-  (identifier) @constant
-  (#match? @constant "^[A-Z][A-Z_]+")
-)
+((identifier) @variable.builtin
+  (#eq? @variable.builtin "self" "this"))
 ```
 
-And this pattern would match key-value pairs where the `value` is an identifier with the same name as the key:
+This pattern would match any identifier that is `self` or `this`.
```

Comment:
Wait, I'm confused. Didn't @ahlinc say that `#eq?` should _not_ behave like `#any-of?`?

Reply:
I think it's time to draw a table somewhere with all predicates and their characteristics.

Reply:
I wrote a footer at the bottom that is basically a  recap